src/loader/level_loader.cpp

```diff
@@ -246,25 +246,26 @@ ActorList preProcessActorDescriptions(
 
       const auto& actor = grid.actorAt(col, row);
       switch (actor.mID) {
-        case 82:
+	    case ActorID::META_Appear_only_in_med_hard_difficulty:
           applyDifficultyMarker(Difficulty::Medium);
           break;
 
-        case 83:
+		case ActorID::META_Appear_only_in_hard_difficulty:
           applyDifficultyMarker(Difficulty::Hard);
           break;
 
-        case 103: // stray tile section marker, ignore
-        case 104: // stray tile section marker, ignore
+		case ActorID::META_Dynamic_geometry_marker_1: // stray tile section marker, ignore
+        case ActorID::META_Dynamic_geometry_marker_2: // stray tile section marker, ignore
           break;
 
-        case 102:
-        case 106:
-        case 116:
-        case 137:
-        case 138:
-        case 142:
-        case 143:
+		case ActorID::Dynamic_geometry_1_falls_down_and_sinks_into_ground:
+		case ActorID::Dynamic_geometry_2_explodes_into_small_parts_flying_away:
+		case ActorID::Dynamic_geometry_3:
+		case ActorID::Dynamic_geometry_4:
+		case ActorID::Dynamic_geometry_5_falls_down_but_remains_intact:
+		// TODO geometry_6??
```

Comment:
@lethal-guitar , not sure, if geometry_6 was left out on purpose.

Reply:
@s-martin nice catch, this is an oversight. Back when I was writing this part of the code, I didn't know that type 6 exists, because none of the levels in the game use it. I discovered it later when reverse-engineering the code for dynamic geometry, but I didn't think of updating this code here.

Feel free to add it for completeness!